alphagov/paas-cf
# Monitor AWS Health for active events, and alert on them

What
----
Our lovely AWS TAM pointed us to the AWS Personal Health Dashboard, which can give us better insights in to ongoing events affecting the health of different AWS services in different regions. 

We want to monitor for those events, and alert on them. This will allow us to do that.

How to review
-------------
Before you start, review PR 242 on account wide terraform and make sure it's applied in the dev account

1. Code review
2. See the [metrics in my environment](https://paas-metrics.andyhunt.dev.cloudpipelineapps.digital/metrics) and look for the `paas_aws_health_active_events` metric
3. Deploy to your env and make sure it works there too.


---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
